ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/bgp-peer-groups-structured-config-2.yml

```diff
@@ -17,15 +17,6 @@ router_bgp:
       maximum_routes: 12000
       send_community: all
       description: Description for ipv4_underlay_peers via structured_config
-    MPLS-OVERLAY-PEERS:
-      type: mpls
-      update_source: Loopback0
-      remote_as: '65001'
-      bfd: true
-      send_community: all
-      maximum_routes: 0
-      route_reflector_client: true
-      description: Description for mpls_overlay_peers via structured_config
```

Comment:
so because this is gone - we are reducing the coverage that we used to have on all the tests for structured config for each peer_group - not sure it is a bad thing as the test is weird anyway.

Reply:
ah but this goes into host 3 actually so probably we are good

Reply:
Correct, the same stuff is moved to the other host, so we have the same coverage. This scenario is not super clean, since we just aim to trigger creation of all peer_groups.